files/lesson1.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+// const myName: string = 'Daria';
```

Comment:
Do not get what do files lesson1-lesson3 mean, why do they need here and if so why do they commented?